main/zoe/guide/contracts/loan.md

```diff
@@ -28,9 +28,8 @@ loaned amount and interest must be of the same (separate) brand.
    drops below `mmr`, liquidation can occur.
 * [`priceAuthority`](/zoe/guide/price-authority.md) - used for getting the current value of
    collateral and setting liquidation triggers.
-* `autoswapInstance` - The running contract instance for an
-   [Autoswap](./autoswap.md) or [AMM](./constantProductAMM.md)
-   installation. The `publicFacet`
+* `AMMInstance` - The running contract instance for 
```

Comment:
This one needs to actually match [the source code](https://github.com/Agoric/agoric-sdk/blob/2ef708a03814ea65d0e6bde4fd0023b25941f985/packages/zoe/src/contracts/loan/index.js#L73) of the contract, so I checked it. It turns out that the loan contract still says `autoswap`.  It can do this because, as I said, autoswap and the newer AMM are mostly compatible.  It's still a bug (though a very low-priority one) that loan still says autoswap. I think it's better to leave this doc matching the contract until we fix it.

Reply:
✔️✔️